gooddata-cn-on-eks/charts.tf

```diff
@@ -115,6 +115,7 @@ resource "helm_release" "gooddata-cn" {
   depends_on = [
     helm_release.pulsar,
     helm_release.kps_crds,
```

Comment:
Update `gooddata-cn.tftpl` to allow setting `podMonitor.enabled` based on `var.enable_kube_prometheus_stack`